tools/importer-rest-api-specs/components/parser/dataworkarounds/workaround_automation_25435.go

```diff
@@ -0,0 +1,37 @@
+package dataworkarounds
+
+import (
+	"fmt"
+
+	"github.com/hashicorp/pandora/tools/importer-rest-api-specs/models"
+)
+
+var _ workaround = workaroundAutomation25434{}
+
```

Comment:
Please add a link to the issue here as well as a brief summary on the issue.

Reply:
added.